_posts/2021-11-29-say-goodbye-to-native-drivers-with-stargate-grpc-api-in-java.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title: "Say Goodbye to Native Drivers with Stargate gRPC API in Java"
+date:   2021-11-29 00:00:00 -0600
+synopsis: "Developers should be able to build highly scalable apps even when they call multilingual and multi-cloud microservices. That’s why the Stargate team recently made a new API generally available to help developers build applications with their preferred language."
+author_info:
+  name:
+    first: "Tomasz"
+    last: "Lelek"
+  picture: "assets/images/stargate-profile.png"
+---
+
+Developers should be able to build highly scalable apps even when they call multilingual and multi-cloud microservices. That’s why the Stargate team recently made a new API generally available to help developers build applications with their preferred language.
+
+The new API is called [gRPC Remote Procedure Call](https://dtsx.io/2Z2nCfU) (gRPC), and we built it on the framework created by Google. In this post, we’ll show you how easy it is to start using this API from the Java ecosystem. We’ll also explain the basic methods exposed to the clients that allow us to interact with the gRPC API.
+
+The setup is simple. The generated code based on protobuf files (“`query.proto`” and “`stargate.proto`”) is shipped with the `grpc-proto` dependency. In your client application, you only need to add two dependencies, the client and a functional channel service provider. In this example we picked netty:
```

Comment:
extra double quotes around `query.proto` and `stargate.proto` don't seem necessary.

Reply:
I thought that felt weird. I'll remove them.